### PR TITLE
CI: Validate on Python 3.6+

### PR DESCRIPTION
**Pull request by amotl**
You never know how you possibly save someone's life with that, running outdated systems and EOL software, but in dear need to get some solid utility installed on the machine or device at hand, mostly in emergency situations.

The README also says:
> Only **Python 3.6+** is supported.

So, here we go.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,6 @@
     "aiofiles",
     "apispec>=1.0.0b1",
     "chardet",
-    "docopt-ng",
```
Comment (tabotkevin) [inline setup.py:24]: Is there a reason why we are removing `docopt-ng`? I know its being used in the `cli.py` file.

**Review by tabotkevin — COMMENTED** [review]
A question..

File: setup.py (hunk 1 above)
Comment (amotl) [reply setup.py:24]: `cli.py` did not exist in this project as I arrived. I asked Kenneth about it, but it seemed he also didn't know. Maybe it fell victim of the reverts he was doing to the code base?

Comment (amotl) [reply setup.py:24]: In this spirit, the dependency can be re-added when we re-add `cli.py` again. Maybe we can find it back in some previous commits or PRs?

Comment (amotl) [reply setup.py:24]: > Maybe we can find it back?

Here we go:

- GH-72
- GH-104

Comment (amotl) [reply setup.py:24]: Let's have a dedicated issue to track this regression.

- GH-547

**Review by tabotkevin — APPROVED** [review]
LGTM!

**Pull request merged**